components/x-privacy-manager/src/__tests__/privacy-manager.test.jsx

```diff
@@ -146,7 +198,10 @@ describe('x-privacy-manager', () => {
 		it('On receiving a response with a non-200 status', () => {
 			const _response = { ok: false, status: 400 };
 			const props = { ...defaultProps, _response };
-			const subject = mount(<BasePrivacyManager {...props} />);
+			const subject = mount(<BasePrivacyManager
+				{...constantProps}
+				{...props}
+			/>);
 
```

Comment:
Sharing this as a purely personal pref, but In general I find that keeping props calculations in the same place involves less context switching between JS/JSX idioms

```tsx
const _response = { ok: false, status: 400 };
const props = { ...defaultProps, ...constantProps, _response };

const subject = mount(<BasePrivacyManager {...props} />);
```